examples/layers/services/isolate.dart

```diff
@@ -0,0 +1,308 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:isolate';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+/// Callback invoked when the progress of a calculation has changed.
```

Comment:
actually since this is just an example, you can probably safely skip the comments for things like typedefs.